sprint0/sprint0/LevelLoader.cs

```diff
@@ -109,7 +109,10 @@ public void AddElement(XmlReader xmlReader)
                 case "Block":
                     string width = xmlReader.GetAttribute("Width");
                     string height = xmlReader.GetAttribute("Height");
-                    if (width != null && height != null)
+                    string sound = xmlReader.GetAttribute("Sound");
```

Comment:
does this line still work if there is no sound attribute?

Reply:
yes

Reply:
yes, it just gets null